R/Lab 1.Rmd

```diff
@@ -0,0 +1,197 @@
+---
```

Comment:
Thank you.  To clarify, using the system.file function, would the following
code function properly?

imagepath <- system.file('Lab1 NWC-DP Images/Figure1.jpg',package='NWCEd')
![**Figure 1: The NWC-DP homepage.**](imagepath)

I used the system.file function to store the file path in the variable
"imagepath" and then reference that path in the line below.  When I "knit
to HTML" to see how it looks, I receive an error message indicating that it
could  not fetch the image path.   Is it safe to assuming that it won't
work until it has been installed in the base folder or could there be
another problem I need to look into?

Thank you for your help.  Also I am now able to commit and push from the
office computer.

Thanks.

On Tue, May 3, 2016 at 11:59 AM, David Blodgett <notifications@github.com>
wrote:

> In R/Lab 1.Rmd
> <https://github.com/dblodgett-usgs/NWCEd/pull/5#discussion_r61926469>:
>
> > @@ -0,0 +1,197 @@
> > +---
>
> The paths to the images should be set using the system.file function.
> https://stat.ethz.ch/R-manual/R-devel/library/base/html/system.file.html
>
> If you put the images in inst/lab1/images/, you would do something like: imagepath
> <- system.file('lab1/images/figure1.jpg',package='NWCEd') in your code.
> This will work because all the stuff in 'inst' gets installed into the base
> folder of the package and system.file('./', package='NWCEd') will return
> the install location of the package.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/dblodgett-usgs/NWCEd/pull/5/files/1b41e74bc8209dc38bef1127b4a9e7cf9235845c#r61926469>
>



-- 
Jake Nelson